.github/workflows/ci.yaml

```diff
@@ -15,9 +15,16 @@ jobs:
           fetch-depth: 0
 
       - uses: actions/setup-dotnet@v4
+        with:
+          source-url: ${{ secrets.NUGET_GSOFTDEV_FEED_URL }}
+        env:
+          NUGET_AUTH_TOKEN: ${{ secrets.GSOFT_NUGET_API_KEY }}
 
       - run: ./Build.ps1
         shell: pwsh
+        env:
```

Comment:
I think the test nuget is being published before we can run the Run-SystemTest.ps1. Shouldn't that test be completed before the publish?

Reply:
Potentially, but since it's only a private feed dedicated for PR package it doesn't have much impact. 

In an ideal world it would be:
- Build
- System-Test
- Deploy package

But since it was already that way it would be out-scope to change this.